cpp/src/slice2js/Gen.cpp

```diff
@@ -2636,10 +2640,10 @@ Slice::Gen::TypeScriptVisitor::visitStructStart(const StructPtr& p)
     {
         _out << sp;
         writeDocCommentFor(dataMember);
-        _out << nl << dataMember->mappedName() << ":" << typeToTsString(dataMember->type(), true) << ";";
```

Comment:
Missing space after the colon here.
On main, we're generating this (which is incorrect):
```
    export class MyStruct
    {
        constructor(MyInt?: number, myString?: string);
        // ...

        MyInt:number;

        myString:string;
    }
```